### ISSUE TITLE
E0401 due to import name clashes with vscode-pylint internal modules

### ISSUE THREAD

**Issue by 2-5** [microsoft/vscode-pylint#224]
I have an `utils` module with an `json_load` module in it. `vscode-pylint`, with default setings (bundled) warns on it's import (INCORRECT):

```python
from utils import json_load
# No name 'json_load' in module 'utils' Pylint(E0611:no-name-in-module)
```

If I `pip install pylint` and then run `pylint` from the terminal, there is no `E0611` warning (CORRECT).

This seems to be caused by `vscode-pylint` adding it's own modules to the path used by `pylint`, and `vscode-pylint` has an `utils` module which overrides mine. This code, which imports a private `vscode-pylint` module has no `E0611` warning (INCORRECT):

```python
# \.vscode\extensions\ms-python.pylint-2022.6.0\bundled\tool\_debug_server.py
from _debug_server import update_sys_path
# no E0611 warning

# .vscode\extensions\ms-python.pylint-2022.6.0\bundled\tool\runner.py
from runner import EXIT_NOW
# no E0611 warning
```

**Comment by fabien-michel** [comment]
I've got this issue each time I import something from a freshly created python module (after pylint server start)
I have to restart Pylint server to have it handle without error the new module import
